npm/private/npm_translate_lock.bzl

```diff
@@ -308,10 +308,11 @@ def _gen_npm_imports(lockfile, attr):
             package = name,
             patch_args = patch_args,
             patches = patches,
-            pnpm_version = pnpm_version,
```

Comment:
Looks like breaking downstream for vendored files, can we avoid that?

Reply:
Shouldn't be breaking as it is an internal variable. Generated .bzl and BUILD files should not be affected.